spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/ChannelFactory.java

```diff
@@ -25,5 +26,24 @@
  */
 public interface ChannelFactory {
 
-	MessageChannel createChannel(String name, Class<?> channelType) throws Exception;
+	/**
+	 * Create a {@link SubscribableChannel} that will be bound to the message channel
```

Comment:
I would change "bound to" to either "bound by" or "bound via"... "the message channel binder" (the binder itself is not a destination for binding, but the agent that binds to some broker/middleware)